add filtering (#1148)
telerik/maui-docs: controls/treedatagrid/filtering/filter-control-template.md

---
title: Filter Control Template
page_title: .NET MAUI TreeDataGrid Documentation - Filter Control Template
description: Review the Telerik .NET MAUI TreeDataGrid Filter Control Template documentation article to learn more about applying custom filter to the DataGrid using FilterControlTemplate property.
position: 2
slug: treedatagrid-filter-control-template
---

# Filter Control Template

The [Telerik UI for .NET MAUI TreeDataGrid]({%slug datagrid-overview%}) inherits the filtering from the DataGrid through the Filtering UI which enables the user to quickly filter the displayed data. 

To customize the built-in Filtering UI for a concrete column, use the `FilterControlTemplate` property of the typed columns (text, boolean, numeric, date, time, and picker columns). 

> In addition, as the [template column]({%slug treedatagrid-columns-template-column%}) does not provide a default Filtering UI, with the `FilterControlTemplate` property you can allow users to filter data in template columns as well.

Reviewer: ```suggestion
As the [template column]({%slug treedatagrid-columns-template-column%}) does not provide a default Filtering UI, to allow users to filter data in template columns, use the `FilterControlTemplate` property.
```
